app/assets/stylesheets/dashboard.scss

```diff
@@ -0,0 +1,281 @@
+// Place all the styles related to the dashboard controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: https://sass-lang.com/
+
+@import "bootstrap";
+
+:root {
+  --background-color: none;
+
+  --full-vertical-height: 100vh;
+  --full-horizontal-width: 100vw;
+
+  --fluid-container-padding: var(--bs-gutter-x, 0.75rem);
+
+  --default-opacity: 0.83;
+
+  --min-width: 300px;
+
+  --main-container-width: clamp(var(--min-width), 100%, 100%);
+  --hrz-list-height: clamp(50px, 100px, 150px);
+  --cat-list-width:  clamp(250px, 400px, 400px);
+  --cat-list-height: clamp(100px, 400px, 400px);
+
+  --item-text-font-size: 12px;
+  --list-item-margin: 10px;
+
+  --resc-prev-size: 100px;
+  --list-item-border: 3px;
+
+  --cat-item-height: 55px;
+  // making sure the margin of the item itself and the border are correctly calculated
+  --cat-item-width: calc(var(--cat-list-width) - calc(var(--list-item-margin) * 2) - calc(var(--list-item-border) * 2));
+
+  --section-padding: 15px;
+  --hrz-list-padding: 0px 20px 0px 20px;
+
+  --cat-inf-padding: 15px;
+}
```

Comment:
Warum nicht https://sass-lang.com/documentation/variables nutzen?

Reply:
ja, hat zwar ein paar schöne features, ist aber aus meiner sicht wie ich es hier strukturiert hab nicht umbedingt nötig, soll ich es trozdem umschreiben?

Reply:
Eine Variable hat viel mehr Vorteile:

>A variable declaration looks a lot like a property declaration: it’s written <variable>: <expression>. Unlike a property, which can only be declared in a style rule or at-rule, variables can be declared anywhere you want. To use a variable, just include it in a value.

Reply:
Ich nehme das mal als ein ja ^^"

Reply:
gut, ich habe mich jetzt kurz damit beschäftigt

Ich werde das denke mal später umschreiben, wenn ich zeit dafür habe.
sass variablen beißen sich sehr mit "clamp", "calc" und vorallem "var(x, y)" in kombination mit den vorherig genannten.